cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -120,7 +120,7 @@ func CreateStaticPodFiles(manifestDir string, cfg *kubeadmapi.InitConfiguration,
 func getAPIServerCommand(cfg *kubeadmapi.InitConfiguration) []string {
 	defaultArguments := map[string]string{
 		"advertise-address":               cfg.LocalAPIEndpoint.AdvertiseAddress,
-		"insecure-port":                   "0",
+		"secure-port":                   "0",
```

Comment:
You can't set `secure-port` to 0. And `secure-port` was set in line 133, so you can just remove this line.